modules/symitriDapRtdProvider.md

```diff
@@ -1,49 +1,88 @@
-### Overview
+---
+layout: page_v2
+title: Symitri DAP Real Time Data Provider Module
+display_name: Symitri DAP Real Time Data Provider Module
+description: Symitri DAP Real Time Data Provider Module
+page_type: module
+module_type: rtd
+module_code : symitriDapRtdProvider
+enable_download : true
+vendor_specific: true
+sidebarType : 1
+---
```

Comment:
This should be in our docs repo as it is labels for our static site generator -> https://github.com/prebid/prebid.github.io. The markdown file here does not connect at all to our docs site and just lives in the PBjs project as reference